### DISCUSSION TITLE
Feature Request: Flipboard Magazine Import

### DISCUSSION THREAD

**Discussion by tomtom215** [karakeep-app/karakeep#532] [Ideas]
Thank you for your work on this app!

Reading your inspiration from using Pocket to the desire to have something self hosted is exactly what my situation except I Have all of my saved bookmarks compiled in [Flipboard](https://about.flipboard.com/) "magazines".

While I absolutely love Flipboard for knowledge compilation, I feel that I could easily lose all of these items if Flipboard goes away, but so far I have not found a great way to export this into a self hosted solution, hence this request. Which is exactly where I would love to leverage Hoarder app.

From what I have figured out so far, the simplest way to import may be by adding ".rss" to the Flipboard magazine url to turn them into a RSS feed, but there are also may also be ways to leverage their APIs via a user auth session cookie to do some scraping that way.

Not only would this allow me to self host my knowledge compiled over the years, it could finally start to use it for some of the Ai features that Hoarder is introducing and more (knowledge graph based RAG).

Thanks for any consideration!

**Comment by kamtschatka** [comment]
please contact flipboard to offer a way to export data. We get all kinds of requests to import data from other tools and it is already overwhelming and it is unlikely that we implement importers for all kinds of formats.
What we are definitely not doing is finding a hacky way to get the data out of another tool that doesn't even offer a way to export the data.

**Reply by tomtom215** [reply]
Understood, thank you
